lib/screens/auth/desktop_auth_screen.dart

```diff
@@ -196,101 +202,261 @@ class DesktopAuthorizationPage extends StatelessWidget {
     );
   }
 
+  String? Function(String?)? get _emailValidator {
+    return (text) {
+      if (text == null || text.isEmpty) {
+        return 'Can\'t be empty';
```

Comment:
Add localization for these strings (Can't be empty, Too short, etc.). 
The same for the lines below.